source/change-streams/tests/unified/change-streams-errors.yml

```diff
@@ -0,0 +1,159 @@
+description: "change-streams errors"
+schemaVersion: "1.0"
```

Comment:
Should this be schema version 1.7? Does 1.7 just include the addition of `ignoreExtraEvents`?

Reply:
Yes to both; fixed.